app/views/chapters/_chapter_management.html.erb

```diff
@@ -7,9 +7,7 @@
       <% end %>
     </li>
   <% end %>
-  
-  <li><%= link_to ts('Edit Chapter'), [:edit, work, chapter] %></li>  
-  <% if work.chapters.count > 1 %>
-    <li><%= link_to ts('Delete Chapter'), [:confirm_delete, work, chapter], :confirm => ts('Are you sure you want to delete this chapter? This will delete all comments on the chapter as well and cannot be undone!') %></li>
+
+  <li><%= link_to ts('Edit Chapter'), [:edit, work, chapter] %></li>
   <% end %>
```

Comment:
I think this end belonged to the logic for the Delete Chapter button, so it will need to go as well.